nginx/nginx.conf

```diff
@@ -1,154 +1,182 @@
 worker_processes  auto;
+pcre_jit on;
+user  nginx;
 
 events {
-  worker_connections  1024;
-  multi_accept        on;
-  use                 epoll;
+    worker_connections  2048;
+    multi_accept        on;
+    use                 epoll;
 }
 
 http {
 
-  upstream app {
-    server web:3000;
-    keepalive 64;
-  }
-
-  proxy_cache_path  /etc/nginx/cache  levels=1:2    keys_zone=STATIC:10m
-    inactive=24h  max_size=1g;
+    upstream app {
+        server web:3000;
+        keepalive 64;
+    }
 
-  server {
-    listen 80 default_server;
+    proxy_cache_path /etc/nginx/cache levels=1:2 keys_zone=STATIC:10m inactive=24h max_size=1g;
 
-    server_name localhost;
+    limit_conn_zone $binary_remote_addr zone=limit_per_ip:10m;
+    limit_conn limit_per_ip 128;
+    limit_req_zone $binary_remote_addr zone=allips:10m rate=150r/s;
+    limit_req zone=allips burst=150 nodelay;
 
-    charset utf-8;
+    server {
+        listen 80 default_server;
 
-    # Redirect all HTTP requests to HTTPS with a 301 Moved Permanently response.
-    return 301 https://$host$request_uri;
-  }
+        charset utf-8;
 
-  server {
-    listen 443 default_server ssl http2;
+        # Redirect all HTTP requests to HTTPS with a 301 Moved Permanently response.
+        return 301 https://$host$request_uri;
+    }
 
-    server_name		localhost;
+    server {
+        listen 443 ssl http2 deferred default_server;
 
-    root /usr/src/app/public;
+        root /usr/src/app/public;
 
-    charset utf-8;
+        charset utf-8;
     
-    ssl_certificate_key /etc/nginx/ssl/yacs.key;
-    ssl_certificate 	/etc/nginx/ssl/chain.cer;
-    ssl_trusted_certificate /etc/nginx/ssl/yacs.cer;
+        ssl_certificate_key /etc/nginx/ssl/privkey.pem;
+        ssl_certificate 	/etc/nginx/ssl/fullchain.pem;
+        ssl_trusted_certificate /etc/nginx/ssl/chain.pem;
```

Comment:
It looks like these are the same file in development. Are they the same file in production as well>